### ISSUE TITLE
Migrate the template to .NET Core 2.0

### ISSUE THREAD

**Issue by petrsvihlik** [Kentico/cloud-boilerplate-net#38]

**Comment by jancerman** [comment]
Currently, when you use the latest .NET Core 2.0 SDK, and try to execute `dotnet build`, you get the following error:

```bash
C:\Program Files\dotnet\sdk\2.0.0\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.DefaultItems.targets(286,5): error : Duplicate 'Content' items were included. The .NET SDK includes 'Conten
t' items from your project directory by default. You can either remove these items from your project file, or set the 'EnableDefaultContentItems' property to 'false' if you want to explici
tly include them in your project file. For more information, see https://aka.ms/sdkimplicititems. The duplicate items were: 'wwwroot\css\ie10-viewport-bug-workaround.css'; 'wwwroot\css\sit
e.css'; 'wwwroot\favicon.ico'; 'wwwroot\js\ie10-viewport-bug-workaround.js'; 'wwwroot\js\site.js' [C:\KC\tools\net-boilerplate\MyWebsite\MyWebsite.csproj]
```

**Comment by petrsvihlik** [comment]
fixed in #50
